packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -800,26 +813,24 @@ export function TransferPanelMain({
   const buttonStyle = useMemo(
     () => ({
       backgroundColor: getBridgeUiConfigForChain(networks.sourceChain.id).color
-        .primary
     }),
     [networks.sourceChain.id]
   )
 
   return (
-    <div className="flex flex-col pb-6">
-      <NetworkContainer network={networks.sourceChain}>
+    <div className="flex flex-col pb-6 lg:gap-y-1">
+      <NetworkContainer bgLogoHeight={140} network={networks.sourceChain}>
```

Comment:
now we do fix height for the logo to not make the container jumpy when error message shows up